kalite/topic_tools/__init__.py

```diff
@@ -259,6 +250,19 @@ def get_content_cache(force=False, annotate=False, language=settings.LANGUAGE_CO
         # Loop through all content items and put thumbnail urls, content urls,
         # and subtitle urls on the content dictionary, and list all languages
         # that the content is available in.
+        contents_folder = os.listdir(settings.CONTENT_ROOT)
+
+        subtitle_langs = {}
+
+        if os.path.exists(i18n.get_srt_path()):
+            for (dirpath, dirnames, filenames) in os.walk(i18n.get_srt_path()):
+                lc = dirpath.split("/")[-2]
```

Comment:
i18n.get_srt_path() returns the base path of the srt folder, so it actually always has subdirectories at least two levels deep:
"en/subtitles/9d8swfjh.srt" is a typical file path - the aim here is to extract the language code from the dirpath ("en/subtitles"), so we have to split("/") and take the penultimate path name.

When it is only listing the contents of the "en" folder rather than the "en/subtitles" folder, this just results in nonsense, but nonsense that never affects us, because the filename is never going to be one that matches the pattern of an srt file.